roles/horizon/templates/etc/openstack-dashboard/local_settings.py

```diff
@@ -109,9 +106,13 @@
 # including on the login form.
 #HORIZON_CONFIG["disable_password_reveal"] = False
 
+HORIZON_CONFIG = {
 {% if horizon.customize %}
-HORIZON_CONFIG["customization_module"] = "horizon-customization.horizon_customization"
+    'customization_module': 'horizon-customization.horizon_customization',
 {% endif %}
+    'user_home': 'openstack_dashboard.views.get_user_home',
```

Comment:
Weird, that's the default in horizon settings.py : https://github.com/openstack/horizon/blob/stable/mitaka/openstack_dashboard/settings.py#L57

So I'm not sure why we have to replace it here.

OOOH!  I bet I know.

Here we're flat out setting `HORIZON_CONFIG` to a new dict, rather than updating values within the dict.

Can we try instead to just do `HORIZON_CONFIG['key'] = 'value'`  for the things we set?